notebooks/hf_vs_tt_t5.ipynb

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
I know we verify completeness often w/ internal notebooks - I thought for those that show parity with HuggingFace or external libraries, we could put those notebooks in the actual repo. Seems like a better way to keep track rather than some Bento notebooks w/ scattered ownership.

Reply:
Sure, but for a quick fix you can right click on expand dots on the top right of this file and select "View file" and it'll give you a notebook view.